emailproxy.py

```diff
@@ -1192,28 +1196,45 @@ def get_oauth2_authorisation_tokens(token_url, redirect_uri, client_id, client_s
 
         if oauth2_flow != 'authorization_code':
             del params['code']  # CCG/ROPCG flows have no code, but we need the scope and (for ROPCG) username+password
-            params['scope'] = oauth2_scope
```

Comment:
Restore this and instead delete if needed in the `password` flow, otherwise the CCG flow will not have a scope value.